plugin/src/main/java/com/iridium/iridiumcore/Item.java

```diff
@@ -93,6 +94,27 @@ public Item(XMaterial material, int slot, int amount, String displayName, String
         this.slot = slot;
     }
 
+    /**
+     * Creates a new item with the provided data.
+     * Used for creating player heads.
+     *
+     * @param material    The material of the item. Specifies the type
+     * @param slot        The slot where this item should be in
+     * @param amount      The amount of this item. Should not be higher than the max stack size
+     * @param displayName The display name of the item with color codes
+     * @param headOwner   The owner of the head
+     * @param lore        The lore of the item, can be empty
+     */
+    public Item(XMaterial material, int slot, int amount, String displayName, String headOwner, List<String> lore, Integer model) {
```

Comment:
U need add some other constructors for items without headdata or slot params etc.